lld/test/ELF/bp-section-orderer-stress.s

```diff
@@ -0,0 +1,104 @@
+# REQUIRES: aarch64
```

Comment:
I wish we didn't have to duplicate this test in ELF and Mach-O, but I can't think of a better solution right now.

Reply:
Duplication like this fine as sharing the test could make updating either port more difficult.